src/app/pages/CpsMap/index.test.jsx

```diff
@@ -1,22 +1,55 @@
+/* eslint-disable react/prop-types */
```

Comment:
Just wanted to check, why is this now necessary in all three test files? Is there any reasonably practical alternative?

Reply:
this was in the originally removed test file but It would be trivial to get rid of this and add  proper prop validations.

Reply:
Hadn't spotted it was in the original test file, apologies.